Dockerfile

```diff
@@ -26,13 +26,10 @@ RUN pip install -e .
 # gunicorn, hypercorn also options https://fastapi.tiangolo.com/deployment/manually/
```

Comment:
there is a new base image that chris and i are trying to promote to get all the images at python 3.9.10. 

FROM renciorg/renci-python-image:v0.0.1

dockerfile at: https://github.com/TranslatorSRI/RENCI-Python-image

example usage at: https://github.com/TranslatorSRI/bl_lookup/blob/dockerfile_updates/Dockerfile

Reply:
Updated the from tag of the image, and removed user creation from this docker file